repository/src/test/java/uk/gov/hmcts/ccd/definition/store/repository/CaseTypeLiteRepositoryTest.java

```diff
@@ -121,8 +121,8 @@ public void checkSQLStatementCounts() {
 
         assertEquals(1, caseTypeLiteEntity.getStates().size());
 
-        // additional query for states and acls (due to eager in StateEntity)
-        SQLStatementCountValidator.assertSelectCount(3);
+        // reduced assertSelectCount to 2 from 3 as acl is now connected lazily in StateEntity
+        SQLStatementCountValidator.assertSelectCount(2);
```

Comment:
The following query, which we do not intend to use, is now dropped.
select
        stateaclen0_.state_id as state_i10_29_0_,
        stateaclen0_.id as id1_29_0_,
        stateaclen0_.id as id1_29_1_,
        stateaclen0_.role_id as role_id9_29_1_,
        stateaclen0_."create" as create2_29_1_,
        stateaclen0_.created_at as created_3_29_1_,
        stateaclen0_.delete as delete4_29_1_,
        stateaclen0_.live_from as live_fro5_29_1_,
        stateaclen0_.live_to as live_to6_29_1_,
        stateaclen0_.read as read7_29_1_,
        stateaclen0_.update as update8_29_1_,
        stateaclen0_.state_id as state_i10_29_1_,
        accessprof1_.id as id2_22_2_,
        accessprof1_.created_at as created_3_22_2_,
        accessprof1_.description as descript4_22_2_,
        accessprof1_.name as name5_22_2_,
        accessprof1_.reference as referenc6_22_2_,
        accessprof1_.security_classification as security7_22_2_,
        accessprof1_.case_type_id as case_typ8_22_2_,
        accessprof1_.dtype as dtype1_22_2_ 
    from
        state_acl stateaclen0_ 
    inner join
        role accessprof1_ 
            on stateaclen0_.role_id=accessprof1_.id 
    where
        stateaclen0_.state_id=?